manager/src/test/java/com/redhat/service/bridge/manager/workers/resources/AbstractWorkerTest.java

```diff
@@ -40,6 +47,23 @@
     @Inject
     BridgeWorker worker;
 
+    @Inject
+    BridgeDAO bridgeDAO;
+
+    @Inject
+    DatabaseManagerUtils databaseManagerUtils;
+
+    @ConfigProperty(name = "event-bridge.resources.worker.max-retries")
+    int maxRetries;
+
+    @ConfigProperty(name = "event-bridge.resources.workers.timeout-seconds")
+    int timeoutSeconds;
+
+    @BeforeEach
+    public void setup() {
+        databaseManagerUtils.cleanUp();
+    }
+
     @Test
     @Transactional
     void workIsCompletedWhenResourceIsNotFound() {
```

Comment:
@manstis I'd like to remove the mocking of the `WorkManager` if we can and use the real implementation. For example, I can see that the following could be re-written as:

```
@Test
    @Transactional
    void workIsCompletedWhenResourceIsNotFound() {
        Bridge bridge = Fixtures.createBridge();
        // It is intetional that we do not persist the Bridge here
        Work work = workManager.schedule(bridge);
        assertThat(workManager.exists(work)).isTrue();

        assertThatThrownBy(() -> worker.handleWork(work)).isInstanceOf(IllegalStateException.class);

        assertThat(workManager.exists(work)).isFalse();
        assertThat(workDAO.findById(work.getId())).isNull();
    }
```

This keeps us as close as possible to our actual flow through the code.